src/CommandLine/Json/DiagnosticGitLabJsonSerializer.cs

```diff
@@ -64,9 +64,11 @@ public static void Serialize(
 #else
             hashBytes = SHA256.HashData(source);
 #endif
+#pragma warning disable CA1872 // Use Convert.ToHexString instead of BitConverter.ToString
```

Comment:
Maybe using `Convert.ToHexString` would be better, wdyt?

Reply:
I believe that this caused issues in older .net versions. Fixing the lint issue caused further downstream issues